login-workflow/src/new-architecture/components/ChangePasswordDialog/ChangePasswordDialog.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { cleanup, fireEvent, render, RenderResult } from '@testing-library/react';
+import { ChangePasswordDialog } from './ChangePasswordDialog';
+import { defaultProps as authContextProps } from '../../contexts/AuthContext/AuthContextProvider.test';
+import { AuthContextProvider } from '../../contexts';
+import { BrowserRouter } from 'react-router-dom';
+import { ChangePasswordDialogProps } from './types';
+
+afterEach(cleanup);
+
+describe('Change Password Dialog tests', () => {
+    let updateFields: any;
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    beforeEach(() => {
+        updateFields = jest.fn();
+    });
+
+    const renderer = (props?: ChangePasswordDialogProps): RenderResult =>
+        render(
+            <AuthContextProvider {...authContextProps}>
+                <BrowserRouter>
+                    <ChangePasswordDialog {...props} />
+                </BrowserRouter>
+            </AuthContextProvider>
+        );
+
+    it('renders without crashing', () => {
+        renderer();
```

Comment:
We are just rendering here, we need assertion to test that it renders without crashing, something like:
expect(something).toMatch(something)